docs/source/user/installing.rst

```diff
@@ -273,33 +273,37 @@ well but were not tested. The use of ``x`` in a version number indicates all
-| 0.49.x | 2020-04-16   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.18     | 0.31.x <= version < 0.33.x   | 9.x               | 2019.5 <= version < 2020.0  |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.48.x | 2020-01-27   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.18     | 0.31.x                       | 8.x               | 2018.0.5 <= version < ?     |
-|        |              |                           |                            |                              | (7.x for ppc64le) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.47.x | 2020-01-02   | 3.5.x <= version < 3.9;   | 1.15 <= version < 1.18     | 0.30.x                       | 8.x               | 2018.0.5 <= version < ?     |
-|        |              | version == 2.7.x          |                            |                              | (7.x for ppc64le) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
++----------++--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| Numba     | Release date | Python                    | NumPy                      | llvmlite                     | LLVM              | TBB                         |
++===========+==============+===========================+============================+==============================+===================+=============================+
+| 0.57.x    | TBC          | 3.7.x <= version < 3.12   | 1.18 <= version < 1.24     | 0.40.x                       | 11.x              | 2021.x                      |
++-----------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| 0.56.x    | TBC          | 3.7.x <= version < 3.11   | 1.18 <= version < 1.23     | 0.39.x                       | 11.x              | 2021.x                      |
++-----------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| 0.55.2    | 2022-05-22   | 3.7.x <= version < 3.11   | 1.18 <= version < 1.23     | 0.38.x                       | 11.x              | 2021.x                      |
++-----------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| 0.55.{0,1}| 2022-01-13   | 3.7.x <= version < 3.11   | 1.18 <= version < 1.22     | 0.38.x                       | 11.x              | 2021.x                      |
```

Comment:
I added separate entries for 0.55.{0,1} and 0.55.2 because 0.55.2 was the first to support NumPy 1.22.